backend/corpus/middleware.py

```diff
@@ -0,0 +1,12 @@
+import logging
+
+
+class RequestLoggerMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+        self.logger = logging.getLogger(__name__)
+
+    def __call__(self, request):
+        self.logger.info(request.__dict__)

```

Comment:
The log output here currently seems to be a Python object. The output has certain fields like: ` 'wsgi.input': < django.core.handlers.wsgi.LimitedStream object at 0x7f4f3e9e66b0 >`. Would it be possible to generate a JSON output for the same? I tried a simple `json.dumps` but that didn't seem to work directly.